src/ipolatev.f90

```diff
@@ -1,3 +1,12 @@
+!> @file
+!! @brief Top-level driver for vector interpolation routine ipolates.
+!! @author Mark Iredell, Kyle Gerheiser
```

Comment:
Don't add your name unless you have authored some of the code in the file. Doxygen reformatting is not code.

Reply:
I authored a lot of the code here when combining them